src/solver/hydro/management/management.h

```diff
@@ -31,10 +31,18 @@
 # include <antares/study/fwd.h>
 # include <antares/mersenne-twister/mersenne-twister.h>
 
-
 namespace Antares
 {
 
+namespace Solver
```

Comment:
Forward declaration

Reply:
I am not a big fan of forward declaration.
I am ok to keep it here, we need to discuss our standards somewhere else